docs/plugin-synopsis.asciidoc.erb

```diff
@@ -18,9 +18,9 @@ Required configuration options:
 
 Available configuration options:
 
-[cols="<,<,<,<m",options="header",]
+[cols="<,<,<,<,<m",options="header",]
 |=======================================================================
-|Setting |Input type|Required|Default value
+|Setting |Input type|Required|Allow dynamic value|Default value
```

Comment:
I dont' know how to document this properly with code annotation. sprintf values are common but not something available for even most settings, I think. It also may not capture other meanings like if a `:validate => :hash` supports sprintf in the values, but not the keys, this may be misleading to users.